Libraries/Opc.Ua.Client/ISession.cs

```diff
@@ -938,7 +938,7 @@ public interface ISession : ISessionClient
         /// <summary>
         /// Sends an additional publish request.
         /// </summary>
-        IAsyncResult BeginPublish(int timeout);
+        void StartPublish(int timeout);
 
```

Comment:
looks good except why the interface change?

Reply:
Maybe add the interface and give it a more descriptive name, e.g. StartQueuingPublishRequests. Thats what it supposed to do.

Reply:
maybe add a flag to ignore GoodOutstandingRequests.